keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/config/ClouddriverConfiguration.kt

```diff
@@ -49,7 +48,7 @@ class ClouddriverConfiguration {
   ):
     CloudDriverService =
     Retrofit.Builder()
-      .addConverterFactory(JacksonConverterFactory.create(objectMapper.disable(FAIL_ON_UNKNOWN_PROPERTIES)))
```

Comment:
This was pretty sketchy since the object mapper is not a prototype bean so this call modifies the existing one.